src/KNXClient.ts

```diff
@@ -1169,7 +1189,7 @@ export default class KNXClient extends TypedEventEmitter<KNXClientEventCallbacks
 		this.setTimer(
 			KNXTimer.GATEWAYDESCRIPTION,
 			() => {},
-			1000 * KNX_CONSTANTS.SEARCH_TIMEOUT,
+			1000 * KNX_CONSTANTS.DEVICE_CONFIGURATION_REQUEST_TIMEOUT,
```

Comment:
could you explain this?

Reply:
Le specifiche KNX non definiscono un valore esatto, ma raccomandano un timeout tipico tra 3 e 10 secondi per garantire un comportamento coerente con i tempi di rete e i dispositivi KNX. La costante DEVICE_CONFIGURATION_REQUEST mi è sembrata la più idonea.